app/templates/10_first_steps.md

```diff
@@ -9,46 +9,52 @@ Lets explore two built-in tools provided to us by Python, `print` and `input`.
 ## <built-in function print\>
 ---
 We can use `print` to print text to the screen.
-A simple syntax to use it is as follows:
+It can be used in the following way:
 
 ```py
 {{hello_world}}
 ```
 
-The parentheses denote that we are calling a function, that is to say that we are invoking its action.
-The quotation marks around 'Hello world' tell Python that it is a string, a data type used to hold human-readable text.
-It is important to indicate that 'Hello world' is a string, otherwise, Python will view this as a syntax error; 
-It would try and interpret 'Hello' and 'world' as labels with special meaning, perhaps user defined variables, and syntactically it would require a comma between these two labels. 
-When you try running this code, experiment with removing the quotation marks and removing words to see how Python tries to tell us what we're doing wrong. 
-'syntax error' may be a little vague to us right now, but getting comfortable with common errors will make it easier to know where to look for help and what to do next time. Overall, though, Python makes the classic 'Hello world' programming tutorial very simple indeed!
+Here, the spelling of print, parentheses after print, and quotation marks around 'Hello world' are all important for this code to work.
+The content inside of the quotation marks, and whether they are single or double quotes, generally do not matter so long as the start and end quotation types match.
```

Comment:
Consider separating clauses: "The content inside of the quotation marks does not matter. Nor, generally, does the use of single versus double quotes - so long as the start and end quotation types match."